.github/workflows/knative-style.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2
         with:
-          fetch-depth: 1
+          fetch-depth: 2
```

Comment:
Why we need two is a bit subtle, so worth a comment.

Reply:
No longer applicable.